Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -655,6 +682,9 @@
                     fprintf('%s\n', ['Warning: Empty image for ' x.SUBJECTS{iSubject} '_ASL_' xASL_num2str(iSess)]);
 				end
 
+                % pGM_here is the main tissue type that is investigated

```

Comment:
We might actually later change the names pGM_here as pMain and pWM_here as pSecond or something like that to avoid a possible confusion with swapping GM and WM.

I'm of course fine with that personally and know the code is correct. It just might be a bit confusing.

Reply:
Yes I thought about this as well. Good idea! Will do this now.